ledger/ledgercore/accountdata.go

```diff
@@ -145,7 +143,7 @@ func (u AccountData) MinBalance(proto *config.ConsensusParams) (res basics.Micro
 
 // IsZero checks if an AccountData value is the same as its zero value.
 func (u AccountData) IsZero() bool {
-	return reflect.DeepEqual(u, AccountData{})
+	return u == AccountData{}
```

Comment:
I also changed this — I believe this was introduced in https://github.com/algorand/go-algorand/pull/126/files#diff-831e406846493c27b22aca1b7482743df94cec8a6957b34059c912ff7996ec28R268 when asset maps were added to AccountData, but now that there are no maps in AccountData we can use regular equality against an empty structure. Or we could change this to a pointer receiver.